modules/tinymce/src/themes/silver/main/ts/ui/menus/item/build/ColorSwatchItem.ts

```diff
@@ -53,3 +53,15 @@ export const renderColorSwatchItem = (spec: Menu.FancyMenuItem, backstage: UiFac
     ]
   };
 };
+
+const getcolorItems = (spec: Menu.FancyMenuItem, backstage: UiFactoryBackstage): Menu.ChoiceMenuItemSpec[] => {
+  if (Type.isNonNullable(spec.initData)) {
+    if (spec.initData.colorselection) {
+      return spec.initData.colorselection.concat(ColorSwatch.getAdditionalColors(false));
```

Comment:
Why are we not using the custom colors here? That definitely seems wrong, as it's supposed to be used for all color swatches if enabled. If you need it to be disabled for specific cases, that should be another `initData` parameter that defaults to adding the custom colors if not set to false.